technical-reports/format/groups.md

```diff
@@ -27,7 +27,7 @@ A file MAY contain many tokens and they MAY be nested arbitrarily in groups like
 
 </aside>
 
-The names of the groups leading to a given token (including that token's name) are that token's _path_, which is a computed property. **It is not specified in the file**, but parsers that conform to this spec MUST be able to expose the path of a token. The above example, therefore, defines 4 design tokens with the following properties:
+  The names of the groups leading to a given token (including that token's name) are that token's _path_, which is a computed property. **It is not specified in the file**, but parsers that conform to this spec MUST be able to expose the path of a token. The above example, therefore, defines 4 design tokens with the following properties:
```

Comment:
Did you mean to indent this paragraph?

```suggestion
The names of the groups leading to a given token (including that token's name) are that token's _path_, which is a computed property. **It is not specified in the file**, but parsers that conform to this spec MUST be able to expose the path of a token. The above example, therefore, defines 4 design tokens with the following properties:
```

Reply:
@kaelig yes, since it's a MD file it doesn't do anything and disappears after conversion to HTML.
Below, a screenshot from the last published draft.

<img width="880" alt="image" src="https://user-images.githubusercontent.com/18498712/201671701-9eb45547-10a5-4036-a9f3-b42a3915e612.png">